uiowa/uiowa
# Allow saving while source edit mode is open.

Resolves #2274 

# Problem
When adding or updating a block with filtered or minimal html text field, if the source edit is open, changes will not be saved.

## Steps to reproduce

From issue: https://github.com/uiowa/uiowa/issues/2274
Add block to section
Create custom block
Edit text in body field
Save block
Edit block
Switch body field to ckeditor source mode
Edit text
Save
Changes are NOT saved!
Once I changed back to non-source mode before saving the changes are saved properly.

## To test
Try the above with the edited JS fix from this branch. Check that updates are now saved.
